app/services/live_lookup/sirsi.rb

```diff
@@ -63,13 +63,14 @@ def initialize(record)
 
       def as_json
         {
-          barcode:,
+          item_id:,
```

Comment:
Are we definitely the only consumers of this response, or should we keep barcode and current_location around (duplicating the data) for backwards compatibility?

Reply:
Either way, I guess we'll find out when we switch over to folio 😬